src/test/java/org/jenkinsci/plugins/github/config/GitHubPluginConfigTest_SEC799.java

```diff
@@ -0,0 +1,65 @@
+package org.jenkinsci.plugins.github.config;
+
+import com.gargoylesoftware.htmlunit.HttpMethod;
+import com.gargoylesoftware.htmlunit.Page;
+import com.gargoylesoftware.htmlunit.WebRequest;
+import hudson.model.Job;
+import hudson.security.GlobalMatrixAuthorizationStrategy;
+import jenkins.model.Jenkins;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+import org.jvnet.hudson.test.JenkinsRule;
+
+import java.net.URL;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.not;
+
+//TODO this class can be merged with GitHubPluginConfigTest after the security fix
+public class GitHubPluginConfigTest_SEC799 {
```

Comment:
please name correctly classes and methods.

Reply:
As a matter of policy, we create new classes for tests added in security fixes to make potential backports of fixes easier. As the TODO comment indicates, this is not intended to be a long lived class.